components/ResilienceResource/ColourBoxSection.js

```diff
@@ -9,7 +9,7 @@ function ColourBoxSection({ sections }) {
     <>
       {sections.map((section) => {
         return (
-          <div key={section.header.title}>
+          <div className="container" key={section.header.title}>
```

Comment:
Is there a reason why you used class container instead of just the Container tag?

Reply:
lol not really. i guess i just found this to be easier